src/Stripe.net/Services/SubscriptionSchedules/SubscriptionScheduleUpdateOptions.cs

```diff
@@ -19,12 +19,6 @@ public class SubscriptionScheduleUpdateOptions : BaseOptions, IHasMetadata
         [JsonProperty("end_behavior")]
         public string EndBehavior { get; set; }
 
-        /// <summary>
-        /// Define the default settings applied to invoices created by this subscription schedule.
-        /// </summary>
-        [JsonProperty("invoice_settings")]
-        public SubscriptionScheduleInvoiceSettingsOptions InvoiceSettings { get; set; }
-
```

Comment:
same here you forgot to remove the class too right?

Reply:
So that one is used in `SubscriptionScheduleDefaultSettingsOptions` and `SubscriptionSchedulePhaseOptions` should that name be different?

Reply:
Yeah it should be `SubscriptionSchedulePhaseInvoiceSettingsOptions` in that case. But not worth renaming for now I'd say